src/common/filter/Expressions.cpp

```diff
@@ -867,7 +867,7 @@ OptVariantType ArithmeticExpression::eval(Getters &getters) const {
         } else if (lv > 0 && rv < 0) {
             return minInt / lv > rv;
         } else if (lv < 0 && rv > 0) {
-            return minInt / lv < rv;
+            return minInt / rv > lv;
```

Comment:
What's the difference from the original

Reply:
The minInt div a negative int(-1) is overflow.